### PR TITLE
MAIN masked input

### PR DESCRIPTION
**Pull request by paschalidi**
screencapture-localhost-3006-2019-01-29-14_22_32

### CODE REVIEW

File: stories/Forms.story.jsx [hunk 1]
```diff
@@ -156,6 +160,15 @@ storiesOf("Forms", module)
                 label="Last name"
                 optional
               />
+              <InputMask
+                mask="1111 1111 1111 1111"
+                id="card"
+                placeholder="Please add your card number"
```
Comment (lanwen) [inline stories/Forms.story.jsx]: better to use our examples, since its our library for medical services

**Comment by lanwen** [comment]
don't really like that this lib will be bundled for every service which wants to use components. How can we avoid that?

**Comment by paschalidi** [comment]
when you say lib you mean the `react-maskedinput`? In general the best approach for a library like this would be that we have the flexibility to add each component separately if we like to.
For example have only `@vivy-components/forms` installed and not the whole package. 

This can be done with lerna easily. 
https://github.com/TerraEclipse/react-stack
look in `modules` file for reference.

This way each component will be released separately that means if for example in the Future that if `Forms` have a breaking change, and a service doesnt want this change it can avoid it by just not installing it. Then it can only install some other packages. 

Le me know if that makes sense

**Comment by paschalidi** [comment]
well another solution is to implement out own masked Input inspired by the `react-maskedinput`.

File: stories/Forms.story.jsx [hunk 2]
```diff
@@ -21,6 +22,9 @@ const rules = {
       message: "Minimum characters is 8"
     }
   ],
+  card: [
+    { type: "string", required: true, message: "Card number is required" }
```
Comment (lanwen) [inline stories/Forms.story.jsx]: wrong error :)

**Review by lanwen — APPROVED** [review]

**Pull request merged**